### ISSUE TITLE
Recent Update: Rules deleted, version no. same

### ISSUE THREAD

**Issue by Rexadev** [safing/portmaster#944]
**What happened**:
1. My PortMaster still shows 0.9.7 Beta but I got new profile feature.
2. All my Global rules got deleted
but
Per App settings show my Global Rules
But they are missing from Global settings page

**Debug Information**:

**Comment by morebears** [comment]
Encountered the same issue today at startup. Beta release channel updated to version 1.0.0. After restarting the PC the app rules are all missing configuration, however they are _mostly_ behaving as if they are configured - e.g. domains that were previously whitelisted are still allowed for outbound communication, but they are not present in the configuration screen.

I uninstalled 1.0.0 (and opted to **keep** the configuration data) and reinstalled the previous stable channel version but the configuration is still in this state, although it seems like fewer applications that were whitelisted still are. I did not have time to collect further information, apologies.
